### PR TITLE
Use materialize css

### PR DESCRIPTION
**Pull request by rogeriochaves**
tl;dr

Check it out: https://build-jqygxtbckq.now.sh/
image

So, I've included materializecss and basically deleted our custom styling from before.
I'm using it from a CDN because:

- Building it together with our css would make webpack config more complex and extractcss plugin is not working well with webpack 2 as far as I tested
- Chances are the user already has it cached on the browser if other site uses it from cdn too
- Github Pages doesn't have http/2, so having different domains may help things to load faster, in parallel
- We can cache it to work offline anyway, with service workers

Somethings I didn't like though:
- Materialize depends on jquery, so we need to load it because of material animations :T
- Materialize css loads the Roboto font, but we use Lato, so we are loading an unused font

Also on this PR:
- I've added an npm script to deploy to *now*, just run `yarn deploy:now` and it will generate a temporary URL that contributors can use to share the app on PRs, just like I did, better to show things than a screenshot!
- Upgrade to webpack 2 final

### CODE REVIEW

**Comment by luismizutani** [comment]
Awesome!!!
